pyvista/core/filters/data_set.py

```diff
@@ -518,6 +518,65 @@ def clip_surface(
         )
         return result
 
+    def slice_implicit(
+        self, implicit_function, generate_triangles=False, contour=False, progress_bar=False
+    ):
+        """Slice a dataset by a VTK implicit function.
+
+        Parameters
+        ----------
+        implicit_function : vtk.vtkImplicitFunction
+            Specify the implicit function to perform the cutting.
+
+        generate_triangles : bool, optional
+            If this is enabled (``False`` by default), the output will
+            be triangles. Otherwise the output will be the intersection
+            polygons. If the cutting function is not a plane, the
+            output will be 3D poygons, which might be nice to look at
+            but hard to compute with downstream.
+
+        contour : bool, optional
```

Comment:
After reading further down, it looks like this was existing for `slice`.  I'm okay keeping but can also consider removing/deprecating.

Reply:
Yeah... this doesn't need to be there. It was originally implemented 4 years ago in https://github.com/pyvista/pyvista/commit/48d1efccdf6793783778731825cf49805165bb23 likely for a support issue at the time. I'd be happy to deprecate this, especially since the contouring parameters cannot be controlled from here